Dockerfile

```diff
@@ -14,12 +14,14 @@ ENV BUILD_RELEASE 1
 
 RUN mkdir -p /opt/tx-processor/scripts
 
-COPY scripts/configure.sh /opt/tx-processor/scripts/configure.sh
+COPY scripts/install-build-tools.sh /opt/tx-processor/scripts/install-build-tools.sh
+COPY scripts/setup-dependencies.sh /opt/tx-processor/scripts/setup-dependencies.sh
 
 # Set working directory
 WORKDIR /opt/tx-processor
 
-RUN scripts/configure.sh
+RUN scripts/install-build-tools.sh
+RUN scripts/setup-dependencies.sh
 
 # Create Build Image
 FROM $BASE_IMAGE AS builder
```

Comment:
In Dockerfile, if the build line includes the flag `-DCMAKE_PREFIX_PATH="prefix"`, the Docker build should succeed

Reply:
Note to @HalosGhost tomorrow morning:

```
/usr/bin/ld: /usr/bin/ld: DWARF error: can't find .debug_ranges section.
../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::Writer**>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIPPN7leveldb6DBImpl6WriterEE8allocateEmPKv[_ZNSt15__new_allocatorIPPN7leveldb6DBImpl6WriterEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<unsigned long>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorImE8allocateEmPKv[_ZNSt15__new_allocatorImE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::CompactionState::Output>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIN7leveldb6DBImpl15CompactionState6OutputEE8allocateEmPKv[_ZNSt15__new_allocatorIN7leveldb6DBImpl15CompactionState6OutputEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<std::pair<int, leveldb::FileMetaData> >::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorISt4pairIiN7leveldb12FileMetaDataEEE8allocateEmPKv[_ZNSt15__new_allocatorISt4pairIiN7leveldb12FileMetaDataEEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::Writer*>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIPN7leveldb6DBImpl6WriterEE8allocateEmPKv[_ZNSt15__new_allocatorIPN7leveldb6DBImpl6WriterEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o):db_impl.cc:(.text._ZNSt15__new_allocatorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE8allocateEmPKv[_ZNSt15__new_allocatorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE8allocateEmPKv]+0x4b): more undefined references to `std::__throw_bad_array_new_length()' follow
collect2: error: ld returned 1 exit status
make[2]: *** [src/parsec/ticket_machine/CMakeFiles/ticket_machined.dir/build.make:94: src/parsec/ticket_machine/ticket_machined] Error 1
make[1]: *** [CMakeFiles/Makefile2:2103: src/parsec/ticket_machine/CMakeFiles/ticket_machined.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 62%] Building CXX object src/parsec/agent/runners/evm/CMakeFiles/evm_runner.dir/serialization.o
[ 63%] Building CXX object src/parsec/agent/runners/evm/CMakeFiles/evm_runner.dir/signature.o
[ 63%] Building CXX object src/parsec/agent/runners/evm/CMakeFiles/evm_runner.dir/util.o
[ 64%] Linking CXX executable runtime_locking_shardd
[ 65%] Building CXX object src/parsec/agent/runners/evm/CMakeFiles/evm_runner.dir/http_server.o
/usr/bin/ld: /usr/bin/ld: DWARF error: can't find .debug_ranges section.
../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::Writer**>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIPPN7leveldb6DBImpl6WriterEE8allocateEmPKv[_ZNSt15__new_allocatorIPPN7leveldb6DBImpl6WriterEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<unsigned long>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorImE8allocateEmPKv[_ZNSt15__new_allocatorImE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::CompactionState::Output>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIN7leveldb6DBImpl15CompactionState6OutputEE8allocateEmPKv[_ZNSt15__new_allocatorIN7leveldb6DBImpl15CompactionState6OutputEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<std::pair<int, leveldb::FileMetaData> >::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorISt4pairIiN7leveldb12FileMetaDataEEE8allocateEmPKv[_ZNSt15__new_allocatorISt4pairIiN7leveldb12FileMetaDataEEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o): in function `std::__new_allocator<leveldb::DBImpl::Writer*>::allocate(unsigned long, void const*)':
db_impl.cc:(.text._ZNSt15__new_allocatorIPN7leveldb6DBImpl6WriterEE8allocateEmPKv[_ZNSt15__new_allocatorIPN7leveldb6DBImpl6WriterEE8allocateEmPKv]+0x4b): undefined reference to `std::__throw_bad_array_new_length()'
/usr/bin/ld: ../../../../prefix/lib/libleveldb.a(db_impl.cc.o):db_impl.cc:(.text._ZNSt15__new_allocatorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE8allocateEmPKv[_ZNSt15__new_allocatorINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE8allocateEmPKv]+0x4b): more undefined references to `std::__throw_bad_array_new_length()' follow
collect2: error: ld returned 1 exit status
make[2]: *** [src/parsec/runtime_locking_shard/CMakeFiles/runtime_locking_shardd.dir/build.make:94: src/parsec/runtime_locking_shard/runtime_locking_shardd] Error 1
make[1]: *** [CMakeFiles/Makefile2:2164: src/parsec/runtime_locking_shard/CMakeFiles/runtime_locking_shardd.dir/all] Error 2
```

Added the above flag to the `./scripts/build.sh` line in `Dockerfile`; got the above errors.